projects/cardiovascular-patient-app/src/environments/environment.logica-open.ts

```diff
@@ -3,6 +3,10 @@ import { fhirclient } from 'fhirclient/lib/types';
 /**
  * A local test environment that uses Logica Sandbox open FHIR server with a hard-coded patient ID
  */
+declare global {
+  interface Window { env: any; }
+}
+
```

Comment:
You can put this in `assets/env.d.ts`. You don't need to repeat it everywhere. You can also add stronger type safety:
```ts
declare global {
  interface Window {
    env: {
      clientId: string;
      appTitle: string;
    };
  }
}
export {};
```